src/data/battle_moves.h

```diff
@@ -10044,7 +10044,7 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
 
     [MOVE_CLANGING_SCALES] =
     {
-        .effect = EFFECT_DEFENSE_DOWN_HIT,
+        .effect = EFFECT_ATTACKER_DEFENSE_DOWN_HIT,
         .power = 110,
         .type = TYPE_DRAGON,
         .accuracy = 100,
```

Comment:
Also change `secondaryEffectChance` to equal 0 since 100% chance is handled by the battle script.